### ISSUE TITLE
Lemmatizing of aegra

### ISSUE THREAD

**Issue by IvyJL** [Hedera-Lang-Learn/hedera#159]
Latin _aegra_ notes Morpheus ambiguities, but doesn’t allow choice between them. Also oddly gives _filia_ as the only choice. 
![F1E37C23-BC48-4D6C-8772-7C139AF944E0](https://user-images.githubusercontent.com/18600875/72205626-3e5c1580-347d-11ea-8906-c1ed38b17858.png)

**Comment by jtauber** [comment]
I'll investigate. Seems weird.

**Comment by jtauber** [comment]
Can confirm raw Morphology API is returning ['aeger', 'aegrum']. I wonder if this is just a data issue.

**Comment by jtauber** [comment]
On a fresh install:

* morpheus lemmatises to an ambiguous ['aeger', 'aegrum']
* aeger is in Logeion but aegrum is not

So I would have thought that, like as with principio, we would have ended up with:

```
aegra
    16867. aeger or aegrum morpheus ambiguity
        <some number>. aeger unwell, ill, sick, diseased, suffering, feeble:
```

but install we still get:

```
aegra
    16867. aeger or aegrum morpheus ambiguity
        539. fīlia, fīliae, f. daughter
```

Adding better debug logging to get to the bottom of this.

**Comment by jtauber** [comment]
Okay, the problem is happening during the Ivy lattice loading, the output of which includes:

```
fīlia	670	fīlia	fīlia, fīliae, f.	daughter	daughter
  created lattice_node 539 fīlia, fīliae, f. daughter
  created morpheus lemma node 452 fīlia

aeger	694	aeger	aeger, aegra, aegrum	unwell, ill, sick, diseased, suffering, feeble
  created lattice_node 539 aeger, aegra, aegrum None
  created morpheus lemma node 453 aeger
```

Notice the same lattice_node id 539 created twice.

**Comment by jtauber** [comment]
Okay, the issue was a bug in the load_ivy_lattice script. Fix incoming...

**Comment by jtauber** [comment]
fixed in b6f125fb6ac0fd143e69c3d44fe9b556b44a01ba
